llmfoundry/models/layers/ffn.py

```diff
@@ -18,6 +20,28 @@
 
 log = logging.getLogger(__name__)
 
+ffn_act_fn_default = {
```

Comment:
nit: signify that it's private and immutable
```suggestion
_FFN_ACT_FN_DEFAULT = {
```

Reply:
similarly for `default_act_fn`